tests/run/sys_path_globbed.srctree

```diff
@@ -0,0 +1,27 @@
+# tag: gh5942
+
+PYTHON setup.py build_ext --inplace
+PYTHON -c "from tglob import mytest; mytest()"
+
+######## tglob.pyx ########
+def mytest():
+    assert True
+
+######## setup.py ########
+import sys
+
+from Cython.Build.Dependencies import cythonize
+from distutils.core import setup, Extension
+
+sys.path.append("[Hello-World]*?")
+
+try:
+    from __builtin__ import reload
+except ImportError:
+    from importlib import reload
+reload(sys)  # Magic
```

Comment:
What exactly is this doing? And why isn't here an explanatory comment?

Reply:
There isn't an explanatory comment because I copied it from a similar test (`reload_ext_module.pyx`) that has no explanatory comment and I have no idea how it works, only that it accomplishes what I need to have done, namely, make Python look at the new `sys.path` I've updated. I banged my head up against the wall trying to figure out a way of setting `sys.path`, which is required for testing, *before* Cythonizing, and this worked, so I used it. However, I'll concede that even if I am bewildered how it works, it merits a comment about what it does.

Reply:
Is there a better way to get Python to use an arbitrary `sys.path` for testing purposes, like a `srctree` command to set the `PYTHONPATH` environment variable before invocation? (I did not get this to work. And simply updating the `sys.path` in `setup.py` doesn't work.) I'd be happy to use something more straightforward. Otherwise, I can add a comment, like `Required to make Python use updated sys.path`.

Reply:
I've fixed the comment per your request. Good to go unless you have a better solution.